src/main.cpp

```diff
@@ -68,7 +68,7 @@ int main(int argc, char **argv){
 
     // The bumper class implementation uses an artificial lrf sensor
     Bumper bumper;
-    double robot_radius = 0.27; // [m]
+    double robot_radius = 0.27;
```

Comment:
Niet noodzakelijk, maar ik zou de eenheid erbij laten staan.
Belangrijker, we moeten de afmetingen nog dubbel checken.